pandas/core/groupby/generic.py

```diff
@@ -748,6 +748,7 @@ def build_codes(lev_codes: np.ndarray) -> np.ndarray:
     def fillna(
         self,
         value: object | ArrayLike | None = None,
+        *,
```

Comment:
We just want to emit a warning, not changing the actual behavior yet

Reply:
Sorry, I didnt get exactly it the first time around. Have added the warnings now.